[Submission] Feedback on essay Feedback Automated tests for infrastructure code #1415

fbjorkman opened the pull request:
Here is the feedback on your essay! I hope it helps :) @amarhod

fbjorkman committed:
Merge pull request #1 from KTH/2021

Update fork

fbjorkman committed:
Added the presentation pdf

fbjorkman committed:
Added Me (Fredrik) as a participant

fbjorkman committed:
Renamed the folder to contain both kth-IDs

fbjorkman committed:
README.md

fbjorkman committed:
Update README.md

fbjorkman committed:
Delete contributions/essay/fbjorkma-adahen directory

fbjorkman committed:
Delete PresentationPrometheus.pdf

fbjorkman committed:
Update and rename contributions/executable-tutorial/chrigu-fbjorkma/README.md to contributions/executable-tutorial/chrigu/README.md

fbjorkman committed:
Create README.md

fbjorkman committed:
Update README.md

fbjorkman committed:
Update README.md

fbjorkman committed:
Update README.md

fbjorkman committed:
Update README.md

fbjorkman committed:
Update README.md

SophieHYe commented:
Hi, 
Thanks for the feedback. Could you please 1) solve the conflicting files and 2) add the final PR of the essay submission(#999 was the initial, not the final). Thanks.

fbjorkman committed:
Update README.md

fbjorkman committed:
Update README.md

Added the final submission PR on the essay

fbjorkman committed:
Merge branch '2021' into feedback

fbjorkman commented:
@SophieHYe The merge conflict have been solved and I have added the PR for the final essay PR submission and also changed the title of this PR to the correct PR number

SophieHYe commented:
Thanks for the update. I am merging your PR.
